server/controllers/userController.js

```diff
@@ -602,219 +238,28 @@ app.post('/account', [
   }
   const rval2 = await pool.query(query2);
 
+  const response = {
+    accountData,
+  };
+  res.locals.response = response;
+  next();
+};
 
-  res.status(200).json(accountData);
-  res.end();
-
-}));
-
-app.post('/transfer/bundle', [
-
-    check('bundle_size','Invaid bundle numbers').isNumeric()
 
-], asyncHandler(async (req, res, next) => {
+/* ________________________________________________________________________
+ * Transfer specific tree tokens between managed accounts
+ * ________________________________________________________________________
+*/
 
+userController.transfer = async (req, res, next) => {
   const errors = validationResult(req);
```

Comment:
I think that the logic for express-validation issues should be lifted out of the controller into the top level where it's specified in the express path specification (or the route could handle this as well for each path)

Reply:
Do you mean right after we do the express-validator checks on the routes before the middleware is called? I think that would probably make most sense to have the router handle for each path, since each of these errors have a slightly different error message.

Reply:
I prefer to follow the principle of encapsulation to move all of the logic of validation to controller, and make the controller/router cleaner, router should be responsible for the path spec, and know nothing about the specific business logic, and let controller deal with all the detail of it.